### PR TITLE
feat: update btcpay-server app to v1.6.3

### PR DESCRIPTION
**Pull request by louneskmt**
- feat: update btcpay-server app to v1.6.3
- lint
- feat: bump version and add release notes

### CODE REVIEW

**Comment by louneskmt** [comment]
Tested and working great on `amd64 testnet`.

File: btcpay-server/docker-compose.yml [hunk 1]
```diff
@@ -8,66 +8,69 @@ services:
...
 
   postgres:
-    image: btcpayserver/postgres:13.6@sha256:dadf0048895a888d88a2dd773dde2f5868c45f74ad37c6d208694df54b590531
+    image: btcpayserver/postgres:13.7@sha256:dadf0048895a888d88a2dd773dde2f5868c45f74ad37c6d208694df54b590531
     # This needs to run as root for migrations to succeed
     # user: "1000:1000"
     restart: on-failure
     # https://github.com/btcpayserver/btcpayserver-docker/commit/a65e7db6851092c75c5ac6c091a5f36ccc5fc26e
-    command: [ "-c", "random_page_cost=1.0" ]
+    command: ["-c", "random_page_cost=1.0", "-c", "shared_preload_libraries=pg_stat_statements"]
```
Comment (nevets963) [inline btcpay-server/docker-compose.yml:68]: Out of interest, how come this `"shared_preload_libraries=pg_stat_statements"` was added? Is BTCPay now reporting DB stats?

Comment (louneskmt) [reply btcpay-server/docker-compose.yml:68]: I've taken this change from here https://github.com/btcpayserver/btcpayserver-docker/commit/a073a5f7b3a010165e882185d6f39c581552c11c. The Dockerfile and entrypoint are available [here](https://github.com/btcpayserver/dockerfile-deps/tree/master/Postgres/13.7). 

The only [results](https://github.com/search?q=org%3Abtcpayserver+pg_stat_statements&type=issues) searching for `pg_stat_statements` in the `btcpayserver` org is https://github.com/btcpayserver/btcpayserver/pull/3688. I guess it was added for debugging purposes.

**Comment by mccolister1017** [comment]
any update on this ?! @lukechilds .

**Pull request merged**